### DISCUSSION TITLE
Badges for Open Source Team members

### DISCUSSION THREAD

**Discussion by MikeMcC399** [corona-warn-app/cwa-website#3279] [General] [answered]
Would it be possible to make Open Source Team members show up here as "Members" or "Collaborators"?

Posts from regular contributors @brianebeling, @MohamedZaghdoudi, @GisoSchroederSAP and @HelTob show with a  "Contributor" badge which is the same as a community member like me who has contributed to the `master` branch. @mtb77, @larswmh and @dsarkar however show with "Member" badges which makes more sense to be able to identify who is here in an "official" capacity.

**Comment by brianebeling** [comment]
Mh, that is weird. For me it shows correctly as Member. It should for anyone that is part of the Corona-Warn-App Team. 🤔 
Is that something you can choose somewhere and I'm just missing it?

**Reply by MikeMcC399** [reply]
![image](https://user-images.githubusercontent.com/66998419/205972158-d938673e-ea39-4e09-b570-3419179f5ab5.png)
 from https://github.com/corona-warn-app/cwa-website/pull/3241#issuecomment-1339565833

You should probably check out the GitHub help https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories and select whichever type of repository this is. It's difficult to tell from outside.

**Comment by brianebeling** [comment]
@MikeMcC399 We wanted to try out discussions anyways, might as well start with this issue, since its not really coding related.

**Reply by brianebeling** [reply]
Interestingly here it shows as "Maintainer"

**Reply by MikeMcC399** [reply]
Discussions definitely have a role to play, although if I'd written "Please change ..." instead of "Would it be possible ..." that would have been an issue, not a discussion.

**Reply by Ein-Tim** [reply]
Related to the discussions feature: https://github.com/corona-warn-app/cwa-wishlist/issues/360

**Reply by brianebeling** [reply]
@MikeMcC399 I agree. Maybe discussions won't be used as much. But if more people post issues regarding the cost of the app it could be worthwhile to link them to a discussion. Hopefully that will prevent more of those issues.

**Reply by brianebeling** [reply]
In regards to enabling discussions on other repos, I'd be careful with that. As discussed in https://github.com/corona-warn-app/cwa-wishlist/issues/360 there are some downsides. For the Wishlist it would make a lot of sense to enable it in my opinion. For something like cwa-server it might not be as helpful. But that depends on the people that actually maintain it. I for one think its helpful to differentiate between general conversation and actionable items. You can also always convert a discussion to an issue if it becomes actionable or vice versa.

**Reply by brianebeling** [reply]
@MikeMcC399 Mind sharing why you disagree with my previous statement? I'm curious why you disagreed.

**Reply by MikeMcC399** [reply]
@brianebeling 
> Mind sharing why you disagree with my previous statement? I'm curious why you disagreed.

I'll put this into a separate new discussion (https://github.com/corona-warn-app/cwa-website/discussions/3282), because this one is about the display of membership badges (which you have already answered! 👍🏻).

**Comment by brianebeling** [comment, marked as answer]
Aha, I found something.

Go to the Org -> People -> Search your username and set it to public
![image](https://user-images.githubusercontent.com/12231155/206005396-9aa378e6-108b-47d7-957d-23fb8d14821d.png)

@MikeMcC399 Thanks for letting me/us know! I will forward it to the other members that are currently active in the repos!

**Reply by MikeMcC399** [reply]
@brianebeling 

Thank you for finding the answer to my question. Your screenshot enabled me to find the instructions in GitHub Help
[Publicizing or hiding organization membership](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership)

It would certainly be helpful for members who are visibly active to make their membership of the [Corona-Warn-App](https://github.com/corona-warn-app) organization public.
